migrator/README.md

```diff
@@ -19,6 +19,26 @@ is operating on, some database instances may be nil.
 A migration can read from any of the databases, make changes to the data or to the datamodel
 (database schema when working with postgres), then persist these changes to the database.
 
+## Migrations and feature flags
+
+The short version is:
+**Do not use feature flags in migration code**.
+The application has to work on the same data model with enabled and disabled feature flags.
+
+If a feature needs a new field (named `new_field` for the purpose of the example) in the stored data,
+this `new_field` needs to be seeded. The data should have an initial seed in the form of a migration
+that populates it from default values, or a mix of existing fields. Once the migration has run,
+the application code should also populate and maintain this field on creation or update of stored objects.
+It is important that this `new_field` should be populated regardless of the fact that the feature is
+activated or not: the feature may have to be temporarily deactivated, and later reactivated, in which case
+the data should still be ready for use by the feature.
```

Comment:
Is this necessarily true?  If I add a `new_field` of type string could I not have the code that uses `new_field` tolerate the empty string and thus I have no need for a migration at all?  Basically wouldn't it be simpler to say that the 0 value of `new_field` must be handled within the code and as such no migration is required unless the field is to be seeded by something else?  Basically I would think in most cases we would want to accept the 0 value to be the default and thus minimizing the need to do anything.